### DISCUSSION TITLE
Missing certificate

### DISCUSSION THREAD

**Discussion by plaffitt** [kubernetes-sigs/kubebuilder#2392] [Support] [answered]
Hello,

I'm trying to implement a mutating admission controller, after generating all the boilerplate and running the controller in a cluster, I get this error : `manager 2021-10-19T16:17:47.371Z    ERROR    setup    problem running manager    {"error": "open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"}`.

What did I missed ?

Thanks

**Comment by camilamacedo86** [comment, marked as answer]
This one seems to be solved here:https://github.com/kubernetes-sigs/kubebuilder/issues/3559

So, closing this out but if you need please raise an issue with your problem
